app/command.go

```diff
@@ -191,48 +192,58 @@ func (a *App) ExecuteCommand(args *model.CommandArgs) (*model.CommandResponse, *
 	return nil, model.NewAppError("command", "api.command.execute_command.not_found.app_error", map[string]interface{}{"Trigger": trigger}, "", http.StatusNotFound)
 }
 
-type mentionMapItem struct {
-	Name string
-	Id   string
-}
-
 // mentionsToTeamMembers returns all the @ mentions found in message that
 // belong to users in the specified team, linking them to their users
 func (a *App) mentionsToTeamMembers(message, teamId string) model.UserMentionMap {
+	type mentionMapItem struct {
+		Name string
+		Id   string
+	}
+
 	possibleMentions := model.PossibleAtMentions(message)
 	mentionChan := make(chan *mentionMapItem, len(possibleMentions))
 
+	var wg sync.WaitGroup
 	for _, mention := range possibleMentions {
+		wg.Add(1)
 		go func(mention string) {
+			defer wg.Done()
 			user, err := a.Srv.Store.User().GetByUsername(mention)
+
+			if err != nil && err.StatusCode != http.StatusNotFound {
+				return
```

Comment:
Suggest we `mlog.Warn` here just to avoid burying the error.

Reply:
Done in https://github.com/mattermost/mattermost-server/pull/13762/commits/9c01498c3cde0eac0ad1882bbf9f66459083f19b.